server/sonar-web/src/main/js/apps/projects/store/utils.js

```diff
@@ -136,6 +151,14 @@ export const convertToFilter = (query, isFavorite) => {
     conditions.push(convertIssuesRating('sqale_rating', query['maintainability']));
   }
 
+  if (query['language'] != null) {
+    if (!Array.isArray(query['language']) || query['language'].length < 2) {
```

Comment:
They proposed both on the api doc, so I thought maybe there are some performance improvements server side using the `=` syntax for 1 element.
But I think I'll still need to check if 'query['language']` is an array or not so I'm not sure I can completely remove the condition.